### PR TITLE
update containers to es6

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

**Comment by Wardormeur** [comment]
According to doc : http://chaijs.com/api/assert/#method_exists
so exists should be a method, not a value.

**Comment by butlerx** [comment]
are we sure this is not http://chaijs.com/api/bdd/#method_exist

File: lib/organisations/controllers/userOrg/create/index.spec.js [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
-    createOrg(payload, function (err, val) {
-      expect(err).to.not.exists;
+    createOrg(payload, (err, {id, userId, orgId}) => {
+      expect(err).to.not.exist;
       expect(senecaStub.export).to.have.been.calledOnce;
-      expect(exportMock.userOrg.save).to.have.been.calledWith(omit(payload, 'ctrl')).once;
-      expect(val.id).to.equal(42);
-      expect(val.userId).to.equal(payload.userOrg.userId);
-      expect(val.orgId).to.equal(payload.userOrg.orgId);
+      expect(exportMock.userOrg.save).to.have.been.calledWith(omit(payload, 'ctrl')).any;
```
Comment (DanielBrierton) [inline lib/organisations/controllers/userOrg/create/index.spec.js]: Pretty sure you don't need `.any` here

**Review by DanielBrierton — CHANGES_REQUESTED** [review]
Main issue is just to align with cp-zen-frontend, so remove spaced out object keys, and require brackets around arrow function params that aren't single line expressions.

File: lib/organisations/controllers/org/list/validation.js [hunk 2]
```diff
@@ -1,4 +1 @@
-var Joi = require('joi');
-module.exports = function (definition) {
-  return {};
-};
+module.exports = () => ({});
```
Comment (DanielBrierton) [inline lib/organisations/controllers/org/list/validation.js:1]: 😂

File: lib/organisations/controllers/org/load/validation.js [hunk 3]
```diff
@@ -1,4 +1 @@
-var Joi = require('joi');
-module.exports = function (definition) {
-  return {};
-};
+module.exports = () => ({});
```
Comment (DanielBrierton) [inline lib/organisations/controllers/org/load/validation.js:1]: 😂 😂 😂 😂 😂 😂 😂 😂 😂  (@Wardormeur)

**Review by DanielBrierton — APPROVED** [review]

**Pull request merged**